src/components/ForgotPassword.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import "./App.css"
+
+export const ForgotPassword = () => {
+    const [Email,setEmail]=useState("");
```

Comment:
const [email, setEmail] = useState("");

The first variable should be lowercase. Please review the React useState documentation: https://react.dev/reference/react/useState#usestate